priv/repo/migrations/20191126115123_create_tables.exs

```diff
@@ -0,0 +1,16 @@
+
+defmodule Crudry.Repo.Migrations.CreateTables do
```

Comment:
Do we need postgres and migrations to test the changeset? I think an embedded schema could be used instead